calc.js

```diff
@@ -1,22 +1,17 @@
-var numField1 = document.getElementById('numField1');
-var numField2 = document.getElementById('numField2');
+var numerator = document.getElementById('numField1');
+var denominator = document.getElementById('numField2');
 var resultField = document.getElementById('resultField');
 var form = document.getElementById('xIsWhatPercentOfY');
 
 
 form.addEventListener('submit', function(event) {
-    if (!numField1.value || !numField2.value) {
+    event.preventDefault();
+    if (!numerator.value || !denominator.value) {
         alert("enter values you loser");
     }
-    else {
-        var x = parseFloat(numField1.value);
-        var y = parseFloat(numField2.value);
-        
-        var result = x / y;
-        var percent = result * 100;
-        
-        resultField.innerText = "Answer " + percent + "%";
-        event.preventDefault();
+    else {    
+        var percent = numerator.value / denominator.value * 100;
+        resultField.innerText = percent === NaN ? 'ensure both values are numbers' : `Answer ${percent}%`;
```

Comment:
this is an inline if, otherwise know as a ternary statement, they're really handy for small bits of logic for assigning values, you can also do `thing = value && value.something` to set thing to `value.something` only if value is truthy (i.e. true, not 0, not null, not undefined, not "0", not NaN)
another useful one is `thing = value1 || value2` where thing is value1 unless it's falsy in which case it's value 2